src/wrap.tsx

```diff
@@ -112,7 +112,7 @@ const debugRequests = () => {
 
 const getMount = () => {
   const { portal, changeRoute, history, mount } = getConfig()
-  const { Component, props, responses, path, hasPath, debug } = getOptions()
+  const { Component, props, responses, path, hasPath, debug, state } = getOptions()
```

Comment:
Although the `state` option is going to be used as an argument of the `atPath` method, as put all the options together by using the `getOptions`, it might sound like it's  a more generic state of `wrapito` if we don't specify that this state is actually the `historyState`. What do you guys think about it?

Reply:
It is true that a word like `state` may not be semantic enough when used with all the other options. `historyState` adds some context, I'm changing it 👌